src/actions.jl

```diff
@@ -23,7 +23,7 @@ function POMDPs.actions(pomdp::RockSamplePOMDP{K}, s::RSState) where K
     end
 end
 
-function POMDPs.actions(pomdp::RockSamplePOMDP{K}, b::ParticleCollection) where K
+function POMDPs.actions(pomdp::RockSamplePOMDP{K}, b::Union{AbstractParticleBelief, SparseCat}) where K
```

Comment:
Oh, I should have caught this earlier! I think we should actually make it as general as possible with this
```suggestion
function POMDPs.actions(pomdp::RockSamplePOMDP, b)
```
(Also, no need for `K`)

Reply:
Ok that makes sense